Namespace(withAug=False, AugDirect=20, method_name='SymDiGCN', GPUdevice=2, IsDirectedData=True, dataset='cora_ml/', undirect_dataset='Amazon-Computers', MakeImbalance=True, CustomizeMask=False, seed=100, K=2, heads=8, log_root='../logs/', log_path='test', data_path='../dataset/data/tmp/', epochs=1500, num_filter=2, p_q=0.95, p_inter=0.1, dropout=0.0, debug=False, new_setting=False, layer=2, lr=0.005, l2=0.0005, to_undirected=False, alpha=0.1, randomseed=-1, imb_ratio=100, n_layer=2, feat_dim=64, warmup=5, epoch=900, tau=2, max=False, no_mask=False, gdc='ppr')
CUDA Device Index: 2
01-01-13:54:40
excel_file_path is  FalseAug_SymDiGCN_cora_ml01-01-13:54:40_dir.xlsx
dataset is  cora_ml 
Dataset is  [Data(x=[2995, 2879], edge_index=[2, 8416], y=[2995], train_mask=[2995, 10], val_mask=[2995, 10], test_mask=[2995, 10])] 
Chosen from DirectedData:  True
Num of bidrected edges: 258, total num of edges: 8416
This is directed graph:  True
data_x torch.Size([2995, 2879])
splits 10
Beginning for split:  0 01-13:55:36
Traceback (most recent call last):
  File "/home/qj2004/magnet/src/GNNChangeTrainTest.py", line 648, in <module>
    main(args)
  File "/home/qj2004/magnet/src/GNNChangeTrainTest.py", line 441, in main
    criterion(out[data_train_mask], data_y[data_train_mask]).backward()
  File "/home/qj2004/.conda/envs/myenv/lib/python3.10/site-packages/torch/_tensor.py", line 396, in backward
    torch.autograd.backward(self, gradient, retain_graph, create_graph, inputs=inputs)
  File "/home/qj2004/.conda/envs/myenv/lib/python3.10/site-packages/torch/autograd/__init__.py", line 173, in backward
    Variable._execution_engine.run_backward(  # Calls into the C++ engine to run the backward pass
RuntimeError: Unable to find a valid cuDNN algorithm to run convolution
